lib/sql-string.js

```diff
@@ -193,14 +193,16 @@ SqlString.bufferToString = function(buffer, dialect) {
 };
 
 SqlString.objectToValues = function(object, timeZone) {
-  var values = [];
-  for (var key in object) {
```

Comment:
Your missing the point. No reason to even think about performance if the code is called very little.
The comparison you linked does not compare lodash methods
.
If you want to improve performance for the project your time would be better spent in the read side of the code, the most expensive stuff we have is cartesian product handling.

You don't have to check ownProperty if it's internal objects. You only have to check it if some dumb code might have modified the prototype. (If people use libs that modify Object.prototype i don't know if that should be our responsibility). I agree that littering the code with `hasOwnProp` would be ugly.